apps/src/code-studio/pd/workshop_dashboard/DetailsPanel.jsx

```diff
@@ -28,6 +28,10 @@ export default class DetailsPanel extends React.Component {
     showAdminEditConfirmation: false
   };
 
+  getToday = () => {
+    return new Date();
+  };
+
```

Comment:
I put the declaration of "today" as a prop for WorkshopForm so that the today object is not re-calculated every single time there's a relevant update since that seems unnecessary (plus it simplifies setting up the tests).